src/Service/ResourceService.php

```diff
@@ -35,7 +36,7 @@ public function getAllResources(?string $permission = null, int $cacheLifetime =
 
         $cachedResources = $this->resourceCache->get("resources-$permission", function (CacheItemInterface $cacheItem) use ($cacheLifetime, $permission) {
             $cacheItem->expiresAfter($cacheLifetime);
-            $info = $this->aakResourceRepository->getAllByPermission($permission);
+            $info = $this->aakResourceRepository->getAllByPermission($permission, $this->excludesResources);
```

Comment:
Typo if you wish to fix
```suggestion
            $info = $this->aakResourceRepository->getAllByPermission($permission, $this->excludedResources);
```